### PR TITLE
A new formula function for creating a VImage

### PR DESCRIPTION
**Pull request by shroffk**
A simple formula for creating a VImage.

In the past we had many many scripts to handle channel access images
=imageOf(array_data, height, width, ...)

### CODE REVIEW

**Review by kasemir — APPROVED** [review]

File: core/formula/src/main/java/org/csstudio/apputil/formula/areadetector/ImageOfFunction.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+        int xoffset = Double.valueOf(VTypeHelper.toDouble(args[5])).intValue();
+        int yoffset = Double.valueOf(VTypeHelper.toDouble(args[6])).intValue();
+
+        boolean xreversed = Boolean.parseBoolean(VTypeHelper.toString(args[7]));
+        boolean yreversed = Boolean.parseBoolean(VTypeHelper.toString(args[8]));
+
+
+        Alarm alarm = Alarm.none();
+        Time time = Time.now();
```
Comment (kasemir) [inline core/formula/src/main/java/org/csstudio/apputil/formula/areadetector/ImageOfFunction.java:108]: Maybe there's a way to copy the time stamp from the image data array? That time stamp might be important.

Comment (kasemir) [reply core/formula/src/main/java/org/csstudio/apputil/formula/areadetector/ImageOfFunction.java:108]: same with alarm info.  Admittedly, for the image widget it doesn't much matter, we plot the data and ignore the time stamp. This is more about future usage of `=imageOf(....`

Comment (shroffk) [reply core/formula/src/main/java/org/csstudio/apputil/formula/areadetector/ImageOfFunction.java]: done

**Comment by kasemir** [comment]
Bummer... Should these be removed?
```
import org.epics.pva.data.PVAByteArray;
import org.epics.pva.data.PVADoubleArray;
import org.epics.pva.data.PVAFloatArray;
import org.epics.pva.data.PVAIntArray;
import org.epics.pva.data.PVALongArray;
import org.epics.pva.data.PVAShortArray;
```

They're not used inside the new function, which makes sense because the function deals with VType data, not the underlying PVA... types.
So no need to import `org.epics.pva.data....`.
In fact must not do that, because now I get a build error:
```
[javac] /opt/jenkins-..../phoebus/core/formula/src/main/java/org/csstudio/apputil/formula/areadetector/ImageOfFunction.java:11: error: package org.epics.pva.data does not exist
    [javac] import org.epics.pva.data.PVAByteArray;
```

**Pull request merged**